int64 id
int64 human_action
int64 human_expression

#message file containing 3 int64 discrete values
